src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -580,4 +614,35 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			this._viewModel[property] = {};
 		}
 	}
+
+	private createTelemetryInfo(): ITelemetryEventProperties {
+		let telemetryInfo = {
+			provider: this._provider.providerId,
+			isNewTable: this._tableInfo.isNewTable,
+		};
+		Object.assign(telemetryInfo, this._telemetryInfo);
+		return telemetryInfo;
+	}
+
+	/// 1. 'Add' scenario
```

Comment:
please make this a proper formatted comment for method.

Reply:
what do you mean proper formatted? in /**/ ?